README.md

```diff
@@ -42,6 +42,7 @@ As maintaining Variable groups in UI is tedious, all AKS pipeline variable group
 
 * To update repository with variable groups if changed through UI, run [Get All Script][variablegroups/scripts/get-all-variable-groups.sh] with a Personal Access Token.
 * Pipeline uses [Update Script][variablegroups/scripts/update-variable-groups.sh] to update all variable groups in [Variable Groups][variablegroups/] while building master branch.
-* To create a new environment, run [Create Script][variablegroups/scripts/add-variable-groups.sh] with access token and environment name to create new variable groups. Fill all fields in the generated files with appropriate values before raising a PR.
-* You can get serviceEndpointId  for keyvaults using [Rest API][https://docs.microsoft.com/en-us/rest/api/azure/devops/serviceendpoint/endpoints/get%20service%20endpoints?view=azure-devops-rest-5.0] 
+* To create a new environment, copy and rename [SBOX-AKS-COMMON.json][variablegroups/SBOX-AKS-COMMON.json] ,[SBOX-AKS-Tags.json][variablegroups/SBOX-AKS-Tags.json], [SBOX-AKS-KeyVault.json][variablegroups/SBOX-AKS-KeyVault.json] . Fill all fields with appropriate values before raising a PR.
+* You can get serviceEndpointId  for KeyVault Variable Group using [Rest API][https://docs.microsoft.com/en-us/rest/api/azure/devops/serviceendpoint/endpoints/get%20service%20endpoints?view=azure-devops-rest-5.0] . If you don't have necessary privileges to list service connections, ask someone who has it. 
```

Comment:
could you add a script that takes the name of the service connection and returns the ID?